Source/Core/Core/ConfigLoaders/MovieConfigLoader.h

```diff
@@ -0,0 +1,32 @@
+// Copyright 2016 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <memory>
+
+#include "Common/Config/Config.h"
+
+namespace Movie
+{
+struct DTMHeader;
+}
+
+class MovieConfigLayerLoader : public Config::ConfigLayerLoader
```

Comment:
`final` class. Also is there any reason this one is exposed in the header, unlike several others?